lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -285,6 +285,30 @@ def self.get_or_create_app(params)
         end
       end
 
+      def self.add_app_to_distribution_if_needed(params)
```

Comment:
just to keep it consistent, please rename `add_app_to_distribution_group_if_needed`

Reply:
renamed 👍